Boa.Constrictor/WebDriver/Extensions/AbstractWebPropertyQuestion.cs

```diff
@@ -3,7 +3,7 @@
     /// <summary>
     /// Abstract class for any Web questions that use a Web element locator and a property name.
     /// </summary>
-    public abstract class AbstractWebPropertyQuestion : AbstractWebLocatorQuestion<string>
+    public abstract class AbstractWebPropertyQuestion<TAnswer> : AbstractWebLocatorQuestion<TAnswer>
```

Comment:
Properties will always be strings. Though, I suppose a subclass could do its own casting.

Part of me wants the simpler version without type generics. Part of me is okay to use type generics.

@chasecastaldo, what's your compelling reason to refactor this class to use a type generic?

Reply:
Part of my original with AbstractWebPropertyListQuestion was to hide IEnumerable<string> cast. After removing that class, HtmlAttributeList + CssValueList are now inheriting from AbstractWebPropertyQuestion (IEnumerable<string>)